draft-ietf-tls-tls13.md

```diff
@@ -1395,7 +1395,7 @@ Structure of this message:
        uint8 CipherSuite[2];    /* Cryptographic suite selector */
 
        struct {
-           ProtocolVersion max_supported_version = { 3, 4 };    /* TLS v1.3 */
+           ProtocolVersion legacy_version = { 3, 3 };    /* TLS v1.2 */
```

Comment:
I'd vote for going uint16, as I did in my version of this proposal. One way or another, we really should do a formal registry, as was suggested on-list in the TLS 1.3 vs 2.0 discussion. I'll submit a PR with just that cherry-picked out of the rebranding PR, as it's a good idea to make this easier to follow, regardless of what we call all this.